connectors/agent/connector_record_manager.py

```diff
@@ -30,25 +28,27 @@ async def ensure_connector_records_exist(self, agent_config, connector_name=None
         """
 
         if not self._agent_config_ready(agent_config):
+            logger.debug(
+                "Agent configuration is not ready to create a connector record."
+            )
```

Comment:
Can we update the copy to provide more specific detail on what's going on? How can configuration be not ready? Is it gonna be happening a lot?

Reply:
> Is it gonna be happening a lot?

It should not be happening a lot, perhaps only once the service starts up with default values.  As soon as we receive first checkin the configuration should be "ready"  ... but this is extra check to ensure that the expected key/value pairs are actually there.